src/components/Toast/ToastNotification.module.scss

```diff
@@ -0,0 +1,75 @@
+@keyframes toastEnterAnimationTop {
```

Comment:
there is an animation.scss file. do you think we can make these animations more generic and move them there?

Reply:
will do!

Reply:
still needs to be moved

Reply:
@nathanyoung done